source/sdk/threading/Thread.ooc

```diff
@@ -165,6 +172,23 @@ Mutex: abstract class {
 
 }
 
+MutexUnsafe: class extends Mutex {
+	init: func
+	lock: override func
```

Comment:
So, basically, Fredrik's reply is that "the test is written a little strangely". I guess that means a fix is on the way.